metis/bin/metis_client

```diff
@@ -651,7 +652,8 @@ EOT
               d.strftime("%b %d  %Y") :
               d.strftime("%b %d %H:%M")
           },
-          file[:file_name] || "#{file[:folder_name]}/"
+          file[:file_name] || "#{file[:folder_name]}/",
+          is_file ? (file[:backup_id] ? "archived" : "unarchived") : "n/a"
```

Comment:
Might want to make this not-the-last column since they just seem to be space-separated and file/folder names can contain spaces. Also looks like you need to add columns below to the bucket map and the width computation.